src/services/user/hooks/index.js

```diff
@@ -393,7 +394,12 @@ const User = require('../model');
 
 
 exports.before = {
-	all: [],
+	all: [
+		search(),
+		search({
```

Comment:
why twice?

Reply:
@smyth64

Reply:
@Parnswir @CeEv Sorry that was by mistake.